### ISSUE TITLE
Feature Request:  Add date/time stamp when device last downloaded provisioning config

### ISSUE THREAD

**Issue by bigfishvoice** [fusionpbx/fusionpbx#1699]
It would help with troubleshooting to know when a device last downloaded the provisioning config file.  Would need to have both date and time.

**Comment by kismetgerald** [comment]
Such a feature would definitely be beneficial to me as well.

**Comment by JoinFreeMe** [comment]
Something that searches through the access log's looking for when provisioning link last got a 200 or 404 or using post on the phone itself as well as something to read the content of that file and display it. Unsure how feasible this is but sounds like a useful feature

**Comment by bigfishvoice** [comment]
Maybe even update the database every time the config file is generated instead of having to parse log files.  It doesn't mean that the provisioning actually worked, just that the device requested the file.  I'd be willing to throw a few bucks at it to move it up the list quicker... :)

**Comment by minotaur01** [comment]
So funny story, i actually coded this tonight! Then saw this post! I will merge the changes in the next day or two.

**Comment by minotaur01** [comment]
Code is been submitted, should be functional as soon as mcrane approves it

**Comment by bigfishvoice** [comment]
Awesome, looking forward to it.  Thanks for your help.

**Comment by minotaur01** [comment]
This feature has been merged with the main distribution, you should be able to update and have the new functionality!

**Comment by bigfishvoice** [comment]
Yep, got it about 2 minutes after Mark merged it.  Thanks for working on it.

**Comment by jlcrews** [comment]
I just updated...where would I look to see this last downloaded date/time?

Thanks for sharing this enhancement.
